### DISCUSSION TITLE
The payment of the earned block does not appear

### DISCUSSION THREAD

**Discussion by jftorrente** [Chia-Network/chia-blockchain#8312] [Support]
Win a block with the hash  ca10c1c925b5bfd356fc3db35e1f0cd3e70e03878e6d2e668be4b798c453d197 
Direccion  xch1snzq6k9gkut3mxnknpvds5g8wvggpwagad3u778uvfhvz7ru77zs84tv60
but it does not appear in my wallet
![1](https://user-images.githubusercontent.com/6894582/131922169-4454c003-ec8c-4479-9e4d-97bc3c095ef5.png)
![2](https://user-images.githubusercontent.com/6894582/131922199-795f0611-a9f0-4fc5-a101-61d732fa9d9c.png)

**Comment by loppefaaret** [comment]
could you tell me what your "farmer reward target address" is ?
you can find it under the "Farm" tab, under the 3`...` in the top right of that tab

**Reply by jftorrente** [reply]
![image](https://user-images.githubusercontent.com/6894582/132009996-2754be76-5224-44bb-a145-7f6f1f23cba3.png)

**Reply by loppefaaret** [reply]
https://xchscan.com/address/xch172wtgxfevzjzrf9pw9jtsfcr6xpwvvxsp46eamt4ma64fa66efzq9enfdg
the transaction hash `ca10c1c925b5bfd356fc3db35e1f0cd3e70e03878e6d2e668be4b798c453d197` refers to block 762893
the above farmer reward target address did not recieve anything around that block, so i can only assume that did _not_ win a block around there, and therefor is not in control of the 1.75 that was in that transaction - the pool did win it, but it was not you that won it i guess ?

**Reply by jftorrente** [reply]
![image](https://user-images.githubusercontent.com/6894582/132025446-61927d16-6376-4218-ba60-42ef2b876eef.png)
 In the pool I appear as the winner of that block

![image](https://user-images.githubusercontent.com/6894582/132025664-218385e3-454a-416a-b5f5-13e13cfafd44.png)

**Reply by loppefaaret** [reply]
spacepool is saying you are the winner of block 762887 - with a farming reward target address in your client of `xch172wtgxfevzjzrf9pw9jtsfcr6xpwvvxsp46eamt4ma64fa66efzq9enfdg` that does not add up with the data in the explorer, they are saying that farmer reward was directed to `xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p` on both:
https://xchscan.com/blocks/762887
https://www.chiaexplorer.com/blockchain/block/0x09d23c01340696f231be5793b876fcc343d550e038e818130905a3edef5f2a06

if you look up the other block they say you have won, it shows correctly your farmer reward target address on block 807150:
https://xchscan.com/blocks/807150
https://www.chiaexplorer.com/blockchain/block/0xf460fe23681b7c702f97e1984cba9586cce7af5f9854007c3ef11f6bbd2901a0

**Reply by jftorrente** [reply]
When checking because I did not have the payment of the block I realized that the address was wrong and then I corrected it, so the new block if it appears

![image](https://user-images.githubusercontent.com/6894582/132028158-12458bbd-a5ad-4e54-9658-75eb52eddbed.png)


Nothing can be done anymore??

**Reply by loppefaaret** [reply]
did you ever had other keys imported in the client, and then removed them ? if you got the 24 words for old keys, you could try and import them, and see if the wallet in there has the missing 0.25 - but i doubt it, the farmer of 762887 is seemingly poolharvest.io

**Reply by jftorrente** [reply]
Previously that contract had it in poolharvest, I change it again to see if it appears like this?

**Reply by loppefaaret** [reply]
ah, no, dont change it back, unless that is the pool you want to be on - unless you got another key to import in your client, those coins are out of your control, unfortunately

**Reply by jftorrente** [reply]
Thanks

**Comment by BrandtH22** [comment]
@jftorrente , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
